src/pages/tickets/org/orgHeader/OrgHeaderStyles.tsx

```diff
@@ -129,6 +130,9 @@ export const CompanyLabel = styled.p`
   font-size: 28px;
   font-style: normal;
   font-weight: 700;
+  margin-top: 10px;
```

Comment:
@MirzaHanan, minor change in your code: `margin: 2px`. Please check this output.
Check this output [Link](https://i.imgur.com/6ZDoooz.png)

Reply:
Sure, I've made the change to margin: 2px as suggested. I'll check the output now and follow up if I have any questions or further updates. Thanks for the guidance!